tests/lib/Content/Form/Provider/AbstractGroupedContentFormFieldsProviderTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\ContentForms\Content\Form\Provider;
+
+use Ibexa\Contracts\ContentForms\Data\Content\FieldData;
+use Ibexa\Contracts\Core\Repository\Values\Content\Field;
+use Ibexa\Core\FieldType\TextLine\Value;
+use Ibexa\Core\Helper\FieldsGroups\FieldsGroupsList;
+use Ibexa\Core\Repository\Values\ContentType\FieldDefinition;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Form\FormInterface;
+
+abstract class AbstractGroupedContentFormFieldsProviderTest extends TestCase
+{
+    protected function getFieldsGroupsListMock(): FieldsGroupsList
+    {
+        $mock = $this->createMock(FieldsGroupsList::class);
+        $matcher = $this->exactly(3);
```

Comment:
We do have team preference to call static methods statically only.
```suggestion
        $matcher = self::exactly(3);
```